Runtime/Interaction/SnapZone.cs

```diff
@@ -151,9 +149,46 @@ private void OnDrawGizmos()
 
         protected virtual void Update()
         {
-            if (socketActive && selectTarget == null)
+            if (showInteractableHoverMeshes)
+            {
+                showInteractableHoverMeshes = false;
+            }
```

Comment:
What is the purpose? Do we want to intentionally not show the dynamic validation highlight?

Reply:
We have highlight as a child object now. It will zedfight with hover mesh.

Reply:
But, does the child highlight changes material when an object hovers the snap zone? Or disappears? 
What happens with other valid objects that have different meshes than the highlight object?